tests/test_examples.py

```diff
@@ -159,88 +112,89 @@ def test_cv_examples(self):
         self.one_complete_example("complete_cv_example.py", False, cv_path, special_strings)
 
 
+@mock.patch.dict(os.environ, {"USE_MOCKED_DATALOADERS": "1"})
 class FeatureExamplesTests(TempDirTestCase):
     clear_on_setup = False
 
-    @mock.patch("checkpointing.get_dataloaders", mocked_dataloaders)
+    @classmethod
+    def setUpClass(cls):
+        super(FeatureExamplesTests, cls).setUpClass()
```

Comment:
```suggestion
        super().setUpClass()
```
We're not in Python 2 ;-)